src/app/components/charts/chart.service.js

```diff
@@ -335,12 +396,17 @@ angular.module('transmartBaseUi')
                     .y(height * set.legH)
                     .itemHeight(4 + 6 * set.min/300)
```

Comment:
Could you make the hard-coded values as 'constant. I know that there's no such a thing in js but just to make it clear what are these number means. 

e.g.
 
var BASE_HEIGHT = 10;
var MIN_CHART_SIZE = 300;

of in a set of obj

var obj  =  {
  CONST_1 = 1,
  CONST_2 = 2
}